ntc_templates/templates/alcatel_aos_show_port-security.textfsm

```diff
@@ -0,0 +1,24 @@
+Value PORT ((\d+\/?)+)
+Value OPERATION_MODE (\S+)
+Value ADMIN_STATE (\S+)
+Value MAX_MAC_BRIDGED (\d+)
+Value TRAP_THRESHOLD (\S+)
+Value MAX_MAC_FILTERED (\d+)
+Value VIOLATION_TYPE (\S+)
+Value VIOLATION_MAC (\S+)
+Value PKT_RELAY (\S+)
+
+
+Start
+  ^\s*Port:\s*${PORT}\s*$$
+  ^\s*Admin-State\s*:\s*${ADMIN_STATE},$$
+  ^\s*Operation\s*Mode\s*:\s*${OPERATION_MODE},$$
+  ^\s*Max\s*MAC\s*bridged\s*:\s*${MAX_MAC_BRIDGED},$$
+  ^\s*Trap\s*Threshold\s*:\s*${TRAP_THRESHOLD},$$
+  ^\s*Max\s*MAC\s*filtered\s*:\s*${MAX_MAC_FILTERED},$$
+  ^\s*Violation\s*:\s*${VIOLATION_TYPE},$$
+  ^\s*Violating\s*MAC\s*:\s*${VIOLATION_MAC}\s*$$
+  ^\s*Pkt-Relay\s*:\s*${PKT_RELAY}$$ -> Record
+  ^\s*
+  ^\s*$$
+  ^.*$$ -> Error
```

Comment:
```suggestion
  ^(Legend:)?\s+Mac\s+Address:
  ^\s*[MmAaCc]{3}\s+Address\s+VLAN\s+TYPE\s*$$
  ^\s*MAC\s+VLAN\s+MAC\s+TYPE\s+OPERATION\s*$$
  ^\s*-+\+
  ^\s*([A-Fa-f0-9]{2}:){5}[A-Fa-f0-9]
  ^\s*$$
  ^. -> Error
```

We don't want to just catch every line that starts with a space, since that is most lines.